python-build/action.yml

```
@@ -0,0 +1,64 @@
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Run flake8 linter
+      shell: bash
+      run: poetry run flake8 .
+
+    - name: Run unittest
+      shell: bash
+      run: | 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)